data/conversations.txt

```diff
@@ -10,31 +10,33 @@
+	`	The pen is heavy as lead but glides almost without friction as you print your initials on the bottom of page after page of ominous legal documents and sign several dozen statements, notices, and agreements. The banker, a balding middle-aged man in a suit that you suspect costs more than the spaceship you are about to purchase, nimbly flips through this dense river of legalese, uttering a curt, "Sign here. Now here," as he points to each page. He moves swiftly, mechanically.`
 	`	The only moment when he becomes truly animated is when explaining the Truth in Lending statement to you. "You are borrowing 480,000 credits," he says, "to be repaid over the course of one year. Your daily interest rate is 0.4%, which means that your daily payments are 2,503 credits, and by the end of the year you will have paid... 434,000 credits in interest. Sign here." He grins.`
 	`	You sign your name one final time:`
 	name
-	`	The elevator is so well-tuned that you do not even realize it is moving until it has deposited you back in the lobby. But as you leave the bank, you are smiling. This crazy adventure suddenly feels real to you. You are going to do it. You are finally going to get off this planet.`
-	`	Compared to the bank, you feel much more at home in the shipyard, walking among the rusted out hulks and newer ships that gleam in the sunlight. You smell grease and dirt and rocket fuel; wonderful smells. There are three ship models within your price range. Which one you choose will determine your future...`
+	`	The elevator is so well-tuned that you do not even realize it is moving until it has deposited you back in the lobby. You bounce out of its metal chassis. A smile, ear to ear, carries you through the lobby, past the shining doormen, and out into the street. It is not until the end of the block that you finally jump and shout. This crazy adventure suddenly feels real to you. You are going to do it. You are finally going to get off this planet.`
+	`	Compared to the bank, you feel much more at home in the shipyard, walking among the rusted out hulks and newer ships that gleam in the sunlight. You hold hands with the smell of grease and dirt and rocket fuel as you drift towards the mechanic. When you ask her what she has in your price range she laughs. "I get one of you about every week." she explains, her face slick with sweat and engine grease. "Yeah alright let's take a look" With that, she leads you to a section of the shipyard with their more modest selections. As you walk, the ships surrounding you go from pristine, to adequate, to worn in, and the two of you finall end up in front of three seasoned vessels. She clasps her arm around your shoulder, chews, and spits. 
+
+"What'll it be, kid?"`
```

Comment:
Shouldn't be so many newlines in between. Either join this part with the previous line, or put it one line after. Remember to put backticks, also.